projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -172,28 +172,37 @@ class ProjectionRegistrySpec
 
       val testProbe = registerProjection(projectionName, tagNames.toSet)
       testProbe.ignoreMsg {
-        case _ => true
+        case FakeStarting(`tagName001`) => false
+        case _                          => true
       }
 
+      // get FakeStarting message from tagName001
+      expectMsgFromWorker(
+        FakeStarting(tagName001),
+        "do-pause-and-resume-worker-test-expect-starting",
+        testProbe,
+        multiExpectTimeout
+      )
+
       // await until seen as ready
       expectWorkerStatus(projectionName, tagName001, Started)
 
-      enterBarrier("do-pause-and-resume-worker-test-all-nodes-ready-001")
+      enterBarrier("do-pause-and-resume-worker-test-tag001-started")
       runOn(RoleName("node2")) {
         projectionRegistry.stopWorker(coordinates001)
       }
 
       expectWorkerStatus(projectionName, tagName001, Stopped)
-      enterBarrier("do-pause-and-resume-worker-test-all-nodes-ready-002")
+      enterBarrier("do-pause-and-resume-worker-test-tag001-stopped")
 
-      // once the worker is stopped we no longer want to ignore messages in the probe.
-      testProbe.ignoreNoMsg()
```

Comment:
See error report: https://github.com/lagom/lagom/issues/2753

I suspect that tagName004 took way too long to start and we stop ignoring the message before it effectively started. As a result, we got its `FakeStarting` message later on causing the test to fail.